src/shared/gpt.h

```diff
@@ -10,6 +10,26 @@
 /* maximum length of gpt label */
 #define GPT_LABEL_MAX 36
 
+typedef enum {
+        GPT_ROOT,
+        GPT_ROOT_VERITY,
+        GPT_ROOT_VERITY_SIG,
+        GPT_USR,
+        GPT_USR_VERITY,
+        GPT_USR_VERITY_SIG,
+        GPT_ESP,
+        GPT_XBOOTLDR,
+        GPT_SWAP,
+        GPT_HOME,
+        GPT_SRV,
+        GPT_VAR,
+        GPT_TMP,
+        GPT_USER_HOME,
+        GPT_LINUX_GENERIC,
+        _GPT_PARTITION_IDENTIFIER_MAX,
+        _GPT_PARTITION_IDENTIFIER_INVALID = -1,
+} GptPartitionIdentifier;
```

Comment:
hmm, we have a very very similar enum in PartitionDesignator. Shouldn't we make this the same?